Week1/homework/js-exercises/1-hello-world.js

```diff
@@ -0,0 +1,30 @@
+// Exercise 1: Hello world!
+{
+'use strict';
+
+console.log("Exercise 1: Hello world!");
+
+let hwEng = 'Hello World';
+let hwTur = 'Merhaba Dunya';
+let hwGer = 'Hallo Welt';
+let hwFr = 'Bonjour le monde';
+let hwEsp = 'Hola Mundo!';
+let hwGr = 'Γειά σου Κόσμε !';
+let hwRus = 'Привет, мир !';
+let hwDutch = 'Hallo Wereld !';
+let hwIt = 'Ciao mondo !';
+let hwInd = 'namaste duniya !';
```

Comment:
1. Can use `const` instead of `let` here.
2. This seems like a very good use case of using a complex data structure such as arrays or objects. It would help in avoiding creating so many variables for different languages. For example,

```
const languages = {
  english: 'Hello, world',
  nederlands: 'Hallo, wereld',
  // and so on
}
```
or
```
const languages = ['Hello World', 'Hallo Wereld', // and so on];
```